### PR TITLE
Show bug title on mouseover

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/34396

### CODE REVIEW

File: lib/OpenQA/WebAPI/Controller/Test.pm [hunk 1]
```diff
@@ -359,17 +359,19 @@ sub job_next_previous_ajax {
...
-        my (@bugs, @bug_urls, @bug_icons);
+        my (@bugs, @bug_urls, @bug_icons, @bug_titles);
         for my $bug (sort { $b cmp $a } keys %$bugs) {
-            push @bugs,      $bug;
-            push @bug_urls,  $self->bugurl_for($bug);
-            push @bug_icons, $self->bugicon_for($bug, $bugdetails->{$bug});
+            push @bugs,       $bug;
+            push @bug_urls,   $self->bugurl_for($bug);
+            push @bug_icons,  $self->bugicon_for($bug, $bugdetails->{$bug});
+            push @bug_titles, $self->bugtitle_for($bug, $bugdetails->{$bug});
```
Comment (Martchus) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: Yet another array. This should be one array containing of hashrefs.

Comment (asdil12) [reply lib/OpenQA/WebAPI/Controller/Test.pm]: updated patch

File: lib/OpenQA/WebAPI/Controller/Test.pm [hunk 2]
```diff
@@ -359,17 +359,18 @@ sub job_next_previous_ajax {
...
 
-        my (@bugs, @bug_urls, @bug_icons);
+        my @bugs;
         for my $bug (sort { $b cmp $a } keys %$bugs) {
-            push @bugs,      $bug;
-            push @bug_urls,  $self->bugurl_for($bug);
-            push @bug_icons, $self->bugicon_for($bug, $bugdetails->{$bug});
+            push @bugs,
+              {
+                'bug'   => $bug,
```
Comment (Martchus) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: I don't think the Perl-critic check will like the quotes here.

**Comment by asdil12** [comment]
See http://artemis.suse.de/tests/795#next_previous and http://artemis.suse.de/tests/overview?distri=opensuse&version=Tumbleweed&build=20181029

**Pull request merged**